### ISSUE TITLE
please add copyright statement (including years) to source.

### ISSUE THREAD

**Issue by onlyjob** [gilmoreorless/jquery-nearest#25]
`src/jquery.nearest.js` is missing copyright statement. Unfortunately link to license is not sufficient... Thanks.

**Comment by gilmoreorless** [comment]
Out of interest, why is a link to the licence not sufficient? jQuery itself only contains a link to its licence, not the full text: http://code.jquery.com/jquery-2.1.3.js

**Comment by gilmoreorless** [comment]
Out of interest, why is a link to the licence not sufficient? jQuery itself only contains a link to its licence, not the full text: http://code.jquery.com/jquery-2.1.3.js

**Comment by onlyjob** [comment]
IMHO jQuery is not following best practices. For instance they include link to their own site (instead of the http://opensource.org/licenses/MIT ) so one should open the link to make sure it is really what they meant by "MIT" license;

Free Software Foundation considers "MIT License" ambiguous because it refers to different licenses:
 * https://en.wikipedia.org/wiki/MIT_License

In your case only text of the license contains copyright statement. You can include copyright statement to .js file header accompanied by link to the text of the license but the latter is short so it will take only little space. For software which is often embedded to other projects It is good to document licensing terms and copyright clearly.

**Comment by onlyjob** [comment]
Also there is an obvious reason to include license instead of a link: imagine situation when some time later project changed hosting or taken down and bundled copies of script links to dead URL... I've seen many such cases where it is hard to assess license compatibility or even to check whether re-distribution is allowed...

**Comment by gilmoreorless** [comment]
OK, I've added a `LICENSE.txt` file in the root of this repository (and therefore on the documentation site as well).
I've also updated the js file comment headers to include years of copyright and a link to the hosted version of `LICENSE.txt`

In order to keep with the "don't ever modify a release" rules of semver, I've released version 1.3.1 with the new comment headers.

Let me know if there are any problems with this new wording. :)

Cheers,
Gil
